pyaedt/generic/settings.py

```diff
@@ -0,0 +1,528 @@
+        It disables the sat bounding box for AEDT version > 2022.2.
+
+        Returns
+        -------
+        str
+            Aedt version in the form ``"2023.x"``.
+        """
+        return self._aedt_version
+
+    @aedt_version.setter
+    def aedt_version(self, value):
+        self._aedt_version = value
+        if self._aedt_version >= "2023.1":
+            self.disable_bounding_box_sat = True
+
+    @property
+    def edb_dll_path(self):
+        """Get/Set an optional path for Edb Dll.
+
+        Returns
+        -------
+        bool
+        """
+        return self._edb_dll_path
+
+    @edb_dll_path.setter
+    def edb_dll_path(self, value):
+        if os.path.exists(value):
+            self._edb_dll_path = value
+
+    @property
+    def global_log_file_size(self):
+        """Get/Set the global pyaedt log file size in Mbytes. The default value is ``10``.
+
+        Returns
+        -------
+        bool
+        """
+        return self._global_log_file_size
+
+    @global_log_file_size.setter
+    def global_log_file_size(self, value):
+        self._global_log_file_size = value
+
+    @property
+    def enable_global_log_file(self):
+        """Enable/Disable the global pyaedt log file logging in global temp folder. Default is `True`.
+
+        Returns
+        -------
+        bool
+        """
+        return self._enable_global_log_file
+
+    @enable_global_log_file.setter
+    def enable_global_log_file(self, value):
+        self._enable_global_log_file = value
+
+    @property
+    def enable_local_log_file(self):
+        """Enable/Disable the local pyaedt log file logging in projectname.pyaedt project folder. Default is `True`.
+
+        Returns
+        -------
+        bool
+        """
+        return self._enable_local_log_file
+
+    @enable_local_log_file.setter
+    def enable_local_log_file(self, value):
+        self._enable_local_log_file = value
+
+    @property
+    def global_log_file_name(self):
+        """Get/Set the global pyaedt log file path. Default is pyaedt_username.log.
+
+        Returns
+        -------
+        str
+        """
+        return self._global_log_file_name
+
+    @global_log_file_name.setter
+    def global_log_file_name(self, value):
+        self._global_log_file_name = value
+
+    @property
+    def enable_pandas_output(self):
+        """
+        Set/Get a flag to use Pandas to export dict and lists. This applies to Solution data output.
+        If ``True`` the property or method will return a pandas object in CPython environment.
+        Default is ``False``.
+
+        Returns
+        -------
+        bool
+        """
+        return self._enable_pandas_output
+
+    @enable_pandas_output.setter
+    def enable_pandas_output(self, val):
+        self._enable_pandas_output = val
+
+    @property
+    def enable_debug_methods_argument_logger(self):
+        """
+        Set/Get a flag to plot methods argument in debug logger.
+        Default is ``False``.
```

Comment:
```suggestion
       Flag for indicating whether to plot methods argument in the debug logger.
        The default is ``False``.
```
I wasn't sure if "methods" should be in a literal string. Or should it be "... plot a method's arguments..."?